### PR TITLE
Travelling

### PR DESCRIPTION
**Pull request by Carreau**
A few things I did while traveling.

### CODE REVIEW

**Comment by Carreau** [comment]
I'll be travelling a bit tomorrow also, but should be online after to do more detailed PRs.

File: Day-1/IPython Kernel/Cell Magics.ipynb [hunk 1]
```diff
@@ -234,6 +241,13 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the **cell** forme, the expression on the first line is executed only once and act as a setup for the body of the cell. The setup step is not timed. THe body of the cell magic, is executed in a tight loop, and the execution time returned."
```
Comment (takluyver) [inline Day-1/IPython Kernel/Cell Magics.ipynb:247]: forme -> form

File: Day-1/IPython Kernel/Cell Magics.ipynb [hunk 2]
```diff
@@ -211,7 +211,14 @@
...
    "source": [
-    "Timing the execution of code; the 'timeit' magic exists both in line and cell form:"
+    "Timing the execution of code; the `timeit` magic exists both in line and cell form."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the line form on the expression on the **line** is timed by beeing executed in a tight loop a number of time."
```
Comment (takluyver) [inline Day-1/IPython Kernel/Cell Magics.ipynb:221]: beeing -> being

Comment (takluyver) [inline Day-1/IPython Kernel/Cell Magics.ipynb:221]: also,  'number of _times_'

File: Day-1/IPython Kernel/Cell Magics.ipynb [hunk 3]
```diff
@@ -491,6 +511,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Keep in mind that when using teh `%%script` magic, or it's equivalents like `%%python`, `%%ruby`, etc... the namespaces are not shared with the current interpreter."
```
Comment (takluyver) [inline Day-1/IPython Kernel/Cell Magics.ipynb:514]: teh -> the, it's -> its

**Comment by takluyver** [comment]
I guess this may be easier if I merge this and fix up the English. I think someone (@dsblank?) had an extension to use the browser spell checker in the notebook.

**Comment by Carreau** [comment]
Yes, unfortunately I was in a train, without internet connexion, 
Same tomorrow. 

I’ll can try to install that later, or should investigate edition in something like EIN, or ipymd. 
— 
M

**Comment by takluyver** [comment]
OK, I've made a few SPG fixes.

I used to edit Wikipedia semi-regularly, and one pet peeve I picked up is a dislike of the phrase ['Note that'](https://en.wikipedia.org/wiki/Wikipedia:%22Note_that%22_is_unnecessary). You can almost always simply cut it out with no loss of clarity. There are situations where it's useful, but they're not very common.

**Comment by dsblank** [comment]
@takluyver No, we have a Javascript-based spell-checker that everyone should be using :smile_cat: 

http://calicoproject.org/ICalico#Calico_Notebook_Extensions

**Pull request merged**